### PR TITLE
Rework Trento web console section

### PR DESCRIPTION
**Pull request by tomschr**
### PR creator: Are there any relevant issues/feature requests?

https://confluence.suse.com/display/TRNT/2022-08-10+Trento+Documentation:

Split it in two sections:

1. Configuration: where we describe the different configuration options currently available in the console. Particularly:
   * cluster settings (we can reutilize the existing content)
   * tags on overviews (hosts, clusters, sap systems, HANA databases)
2. Use cases:
   * dashboard description (we can reutilize de exiting content)
   * what to do when host status is not green
   * what to do when pacemaker cluster status is not green
   * what to do when database status is not green
   * what to do wen SAP instances status is not green

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+
+    <section xml:id="sec-trento-manage-tags">
+      <title>Managing tags</title>
+      <para>
+       Tags are a means to group specific objects by assigning a purpose, location, owner, or other
+       properties you like. Such objects can be hosts, clusters, databases, or SAP systems.
+       Tags makes it easer to distinguish and show all these different objects to make
+       your lists more readable and searchable. You can use any text you like to create
+       your tags.
+      </para>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1377]:
```suggestion
      <para>
       Tags are a means to label specific objects with a purpose, location, owner, or any other
       property you like. Such objects can be hosts, clusters, databases, or SAP systems.
       Tags make it easer to distinguish and show all these different objects, making
       your lists more readable and searchable. You can use any text you like to create
       your tags except blank spaces and special characters other than + - = . , _ : and @.
      </para>
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+          </step>
+        </procedure>
+        <para>
+          After you have finished the above procedure, the changed host contains one or
+          more tags.
+        </para>
+        <para>
+          The same principle applies to the other entries from the dashboard, be it for <guimenu>Clusters</guimenu>,
+          <guimenu>SAP Systems</guimenu>, or <guimenu>HANA Databases</guimenu>.
+        </para>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1427]:
```suggestion
        <para>
          The same principle applies to the other objects in Trento, be it <guimenu>Clusters</guimenu>,
          <guimenu>SAP Systems</guimenu>, or <guimenu>HANA Databases</guimenu>.
        </para>
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+            <para>
+              Log in to Trento.
+            </para>
+          </step>
+          <step>
+            <para>
+              In the Trento dashboard, click the respective part that you are interested in, be it a host, a cluster, or
+              anything else. For this example, as we show it for hosts, click the <guimenu>Hosts</guimenu>
+              entry.
+            </para>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1446]:
```suggestion
<para>
In the Trento dashboard, go to the overview corresponding to the object for which you want to remove tags. For example, the <guimenu>Hosts</guimenu> overview.
</para>
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+          </step>
+          <step>
+            <para>
+              In the <guimenu>Hosts</guimenu> view, search for your host you want to remove a tag.
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1450]:
```suggestion
              In the <guimenu>Hosts</guimenu> overview, search for the host you want to remove a tag from.
```

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+              or on a different host.
+            </para>
+          </step>
+        </procedure>
+      </section>
+
+      <section xml:id="sec-trento-filter-tags">
+        <title>Filter tags</title>
+        <para>
+          With tags, you can show only those parts you are interested in. For example, if
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1471]:
```suggestion
          With tags, you can show only those objects you are interested in. For example, if
```

File: trento/xml/article_sap_trento.xml [hunk 6]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+          you have created a <literal>drbd</literal> tag for some hosts, you can show only
+          those hosts that have this <literal>drbd</literal> tag assigned.
+        </para>
+        <procedure>
+          <step>
+            <para>
+              In the Trento dashboard, click the respective part you are interested in.
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1478]:
```suggestion
              In the Trento dashboard, go to the respective overview you are interested in.
```

File: trento/xml/article_sap_trento.xml [hunk 7]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+          </step>
+          <step>
+            <para>
+              Make sure you have already assigned some tags. If not, refer to <xref linkend="sec-trento-add-tags"/>.
+            </para>
+          </step>
+          <step>
+            <para>
+              In the second row, click the last pull down menu with the name <guimenu>Filter tags</guimenu>.
+              The pull down menu opens and shows you all of your definied tags.
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1489]:
```suggestion
              The pull down menu opens and shows you all of your defined tags.
```

**Review by abravosuse — CHANGES_REQUESTED** [review]

File: trento/xml/article_sap_trento.xml [hunk 8]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+        <para>In the list, search for a cluster, for example a &hana; cluster.</para>
+      </step>
+      <step>
+        <para>Click the respective cluster name in the name column. The details view opens.</para>
+        <remark>tom 2023-01-16: screenshot</remark>
+        <figure>
+          <title>&pace; cluster details</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata width="70%" fileref="trento-web-pacemaker-cluster-details-view.png"/>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:829]: I accidentally marked the previous comment as resolved but the issue with the screenshot persists:

This screenshot has been take in a small screen. The checks result section should show on the left of the section with the cluster runtime information (provider, SID, fencing type, etc). Both sections should be horizontally aligned.

File: trento/xml/article_sap_trento.xml [hunk 9]
```diff
@@ -794,7 +786,750 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+     dashboard description (we can reutilize the exiting content)
+     - what to do when host status is not green
+     - what to do when pacemaker cluster status is not green
+     - what to do when database status is not green
+     - what to do wen SAP instances status is not green
+  -->
+  <section xml:id="sec-trento-checks">
+    <title>Performing configuration checks</title>
+    <para>
+      One of Trento's main feature is to provide checks for the HANA scale-up performance optimized two-node clustes in your SAP Landscape.
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:808]:
```suggestion
      One of Trento's main features is to provide checks for the HANA scale-up performance optimized two-node clustes in your SAP Landscape.
```

File: trento/xml/article_sap_trento.xml [hunk 10]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+        <figure xml:id="fig-pacemaker-clustersettings">
+          <title>&pace; Cluster Settings</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata width="70%" fileref="trento-web-pacemaker-clustersettings.png"/>
+            </imageobject>
+          </mediaobject>
+        </figure>
+      </step>
+      <step>
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:846]: Still pending

File: trento/xml/article_sap_trento.xml [hunk 11]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+      </itemizedlist>
+
+      <para>
+        The dashboard contains the following states (see <xref linkend="fig-trento-web-home"/>):
+      </para>
+      <figure xml:id="fig-trento-web-home">
+        <title>Dashboard with the global health state</title>
+        <mediaobject>
+          <imageobject>
+            <imagedata width="80%" fileref="trento-web-home.png" />
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1013]: Still pending

File: trento/xml/article_sap_trento.xml [hunk 12]
```diff
@@ -794,7 +786,750 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+        </varlistentry>
+        <varlistentry>
+          <term><guimenu>Critical</guimenu></term>
+          <listitem>
+            <para>
+              Contains the number of critical systems.
+              A system with at least one component in critical (red) status, will be computed
+              as <emphasis>Critical</emphasis> in the global health section.
+              As administrator, these systems are usually broken and need to be repaired.
+            </para>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1043]:
```suggestion
            <para>
              It shows the number of systems with at lease one layer in critical (red) status.
            </para>
```

File: trento/xml/article_sap_trento.xml [hunk 13]
```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+          </step>
+          <step>
+            <para>
+              In the <guimenu>Tags</guimenu> column, click the <guimenu>Add Tag</guimenu> entry.
+            </para>
+          </step>
+          <step>
+            <para>Enter the respective tag and finish with <keycap function="return"/>.</para>
+          </step>
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1433]: Still pending

**Review by abravosuse — CHANGES_REQUESTED** [review]

File: trento/xml/article_sap_trento.xml (hunk 13 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:1433]: Yes, I found out it's actually a problem in our stylesheets. I fixed it in PR openSUSE/suse-xsl#529.

**Comment by tomschr** [comment]
I will look into the open issues tomorrow.

File: trento/xml/article_sap_trento.xml (hunk 11 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:1037]: I'm not entirely sure, so I did the screenshot again and committed it to commit ea60b89. Does that help?

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1037]: Yes, it is fixed now.

**Review by abravosuse — CHANGES_REQUESTED** [review]

File: trento/xml/article_sap_trento.xml [hunk 14]
```diff
@@ -794,7 +786,772 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
...
+        <para>In the list, search for a &hana; cluster.</para>
+      </step>
+      <step>
+        <para>Click the respective cluster name in the name column. The details view opens.</para>
+        <remark>tom 2023-01-16: screenshot</remark>
+        <figure>
+          <title>&pace; cluster details</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata width="80%" fileref="trento-web-pacemaker-cluster-details-view.png"/>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:829]: Yes, please do!

**Pull request merged**